media/book/src/implementing_solver.md

```diff
@@ -1,7 +1,7 @@
 # Implementing a solver
 
 In this section we are going to implement the Landweber solver, which essentially is a special form of gradient descent.
-In iteration \\( k \\), the new parameter vector \\( x_{k+1} \\) is calculated from the previous parameter vector \\( x_k \\) and the gradient at \\( x_k \\) according to the following update rule:
+In iteration \\( k \\), the new parameter vector \\( x\_{k+1} \\) is calculated from the previous parameter vector \\( x_k \\) and the gradient at \\( x_k \\) according to the following update rule:
```

Comment:
My autoformatter made most changes to this file.

Reply:
I wasn't sure if the change in line 4 would render correctly but surprisingly it does.